ql/lineup.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from django.utils.functional import cached_property
+from cached_property import cached_property

```

Comment:
Faktycznie, przeoczyłem. To było chyba z czasów, jak próbowałem wymienić ORM. W requirements go nie umieściłem, więc oczywiście nie pozostało celowo.

Linijka do revertu.